content/docs/introduction/how-billing-works.md

```diff
@@ -21,9 +21,11 @@ Each of Neon's plans includes **Storage**, **Compute**, and **Project** usage al
 These allowances are included in your plan's monthly fee, except for the Free Tier, which is always free. You can find the monthly fees for the [Launch](/docs/introduction/plans#launch) and [Scale](/docs/introduction/plans#scale) plans on our [pricing](https://neon.tech/pricing) page. The [Enterprise](/docs/introduction/plans#enterprise) plan is completely customizable with respect to allowances.
 
 <Admonition type="tip" title="What is a compute hour?">
-A **compute hour** is 1 hour of _active time_ for a compute with 1 vCPU. If you have a compute with .25 vCPU, as you would on the Neon Free Tier, it would take 4 hours of _active time_ to use 1 compute hour. On the other hand, If you have a compute with 4 vCPU, it would only take 15 minutes of _active time_ to use 1 compute hour.
 
-**_**Active time** is the amount of time a compute is active as opposed to being idle due to being suspended due to inactivity. The time that your compute is idle is not counted toward compute usage.
+**Active hours** meausre the amount of time a compute is active as opposed to being idle due to being suspended due to inactivity. The time that your compute is idle is not counted toward compute usage.
```

Comment:
```suggestion
**Active hours** measures the amount of time a compute is active, rather than idle when suspended due to inactivity. The time that your compute is idle is not counted toward compute usage.
```